spec/support/capybara.rb

```diff
@@ -1,7 +1,17 @@
-require "capybara/poltergeist"
-
 Capybara.configure do |config|
   config.match = :prefer_exact
-  config.javascript_driver = :poltergeist
   config.asset_host = "http://#{ENV.fetch('HOST')}"
 end
+
+Capybara.register_driver :headless_chrome do |app|
```

Comment:
Could we use options from https://github.com/teamcapybara/capybara/blob/master/lib/capybara/registrations/drivers.rb#L27-L36 

* `opts.args << '--disable-gpu' if Gem.win_platform?`
* `opts.args << '--disable-site-isolation-trials'`